monai/data/image_writer.py

```diff
@@ -41,7 +42,69 @@
     PILImage, _ = optional_import("PIL.Image")
 
 
-__all__ = ["ImageWriter", "ITKWriter", "NibabelWriter", "PILWriter", "logger"]
+__all__ = [
+    "ImageWriter",
+    "ITKWriter",
+    "NibabelWriter",
+    "PILWriter",
+    "SUPPORTED_WRITERS",
+    "register_writer",
+    "resolve_writer",
+    "logger",
+]
+
+SUPPORTED_WRITERS: Dict = {}
+
+
+def register_writer(ext_name, *im_writer):
+    """
+    Register ``ImageWriter``, so that writing a file with filename extension ``ext_name``
+    could be resolved to a tuple of potentially appropriate ``ImageWriter``.
+    The customised writers could be registered by:
+
+    .. code-block:: python
+
+        from monai.data import image_writer
+        # `MyWriter` must implement `ImageWriter` interface
+        image_writer.register_writer(".nii", MyWriter)
+
+    Args:
+        ext_name: the filename extension of the image.
```

Comment:
Here is arg is `ext_name` while in readers it's `suffixes`, I feel it's better to be consistent?

Thanks.